internal/provider/resources/service_account.go

```diff
@@ -131,7 +131,7 @@ func (r *ServiceAccountResource) Schema(_ context.Context, _ resource.SchemaRequ
 				Description: "Account Role name of the service account",
```

Comment:
Wonder if it's worth mentioning the possible values here:

1. It would tell you the list of possible values.
2. You can use a correct value the first time, instead of putting a wrong one and having it error out later.

It'd show up here:

<img width="644" alt="image" src="https://github.com/user-attachments/assets/eafb8e51-06a4-4cbc-9ee5-cd74e3154f86">

Reply:
```suggestion
				Description: "Account Role name of the service account (one of: Admin, Member, Owner)",
```

Something like that!